### PR TITLE
emphasize named-entities with quotes to reduce the scope of WORDLIST

### PR DESCRIPTION
**Pull request by m7pr**
@pawelru this is the mission to reduce the WORDLIST only to crucial terms. The list is now reduced to 10 items. If you run `spelling::spell_check_package()` you can see where the existing words occur 

Looks like spell-checker recognizes words like `cloneable`, `programmatically` and `reproducibility` as issues but those are legit development words. We can argue if we should put Kahn in quotes or italics in `topological` vignette. Other than that I think we are good to go for the current state. Stating a PR to `filter_panel_refactor@main` as this is the current `main` for now.

```{R}
WORD               FOUND IN
cloneable          ChoicesFilterState.Rd:242
                   DateFilterState.Rd:235
                   DatetimeFilterState.Rd:245
                   DefaultFilteredDataset.Rd:252
                   DFFilterStates.Rd:206
                   EmptyFilterState.Rd:146
                   FilteredData.Rd:940
                   FilteredDataset.Rd:395
                   FilterPanelAPI.Rd:175
                   FilterState.Rd:289
                   FilterStateExpr.Rd:334
                   FilterStates.Rd:347
                   LogicalFilterState.Rd:222
                   MAEFilteredDataset.Rd:221
                   MAEFilterStates.Rd:107
                   MatrixFilterStates.Rd:79
                   RangeFilterState.Rd:262
                   SEFilterStates.Rd:149
Forkers            README.md:42,52,54
funder             teal.slice-package.Rd:26
Github             README.md:30
Hoffmann           teal.slice-package.Rd:26
Kahn               topological_sort.Rd:17
programmatically   NEWS.md:21
                   teal-slice.Rmd:95
repo               README.md:9,15,50,54
reproducibility    FilteredData.Rd:174,322,518,526
                   init_filtered_data.Rd:24
                   NEWS.md:93
UI                 DefaultFilteredDataset.Rd:214,216,230
                   FilteredData.Rd:738,782,809,830,854,875,884,922
                   FilteredDataset.Rd:298,300,315,333,345,347,361,383
                   FilterState.Rd:255,265
                   FilterStateExpr.Rd:313,323
                   FilterStates.Rd:179,256,258,301,335
                   MAEFilteredDataset.Rd:185,187,201
                   MAEFilterStates.Rd:86
                   SEFilterStates.Rd:100,137
                   filter-panel.Rmd:151
                   teal-slice.Rmd:48
```

### CODE REVIEW

File: inst/WORDLIST [hunk 1]
```diff
@@ -1,27 +1,10 @@
...
-coercible
-dataname
-datanames
-DDL
-dropdown
-FilterState
-flexbox
 Forkers
-funder
 Github
```
Comment (pawelru) [inline inst/WORDLIST:2]: How about this guy? Can we change it into GitHub?

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -42,9 +42,9 @@ For new feature requests, please elaborate on the context and the benefit the fe
 
 ## Pull requests
 
-### Github Flow
+### GitHub Flow
 
-This repository uses the [Github Flow](https://docs.github.com/en/get-started/quickstart/github-flow) model for collaboration. To submit a pull request:
+This repository uses the [GitHub Flow](https://docs.github.com/en/get-started/quickstart/github-flow) model for collaboration. To submit a pull request:
```
Comment (pawelru) [inline .github/CONTRIBUTING.md:47]: This particular file has been copied from `r.pkg.template` repo into all of other repos. If you want to propose changes there I would open a PR and then ask IDR to propagate.

File: inst/WORDLIST (hunk 1 above)
Comment (m7pr) [reply inst/WORDLIST]: Totally! My obvious oversight! Fixed https://github.com/insightsengineering/teal.slice/pull/331/commits/b675c07cd36175624741586cfc337fd0cc7ec1e2

**Review by pawelru — APPROVED** [review]
This looks good. Thanks!

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (m7pr) [reply .github/CONTRIBUTING.md:47]: Thanks. Didn't know about it. Opened a PR here https://github.com/insightsengineering/r.pkg.template/pull/157

**Comment by m7pr** [comment]
Decided to clean up a bit more, and this is the final outcome - 8 words

```{R}
  WORD               FOUND IN
cloneable          ChoicesFilterState.Rd:242
                   DateFilterState.Rd:235
                   DatetimeFilterState.Rd:245
                   DefaultFilteredDataset.Rd:252
                   DFFilterStates.Rd:206
                   EmptyFilterState.Rd:146
                   FilteredData.Rd:940
                   FilteredDataset.Rd:395
                   FilterPanelAPI.Rd:175
                   FilterState.Rd:289
                   FilterStateExpr.Rd:334
                   FilterStates.Rd:347
                   LogicalFilterState.Rd:222
                   MAEFilteredDataset.Rd:221
                   MAEFilterStates.Rd:107
                   MatrixFilterStates.Rd:79
                   RangeFilterState.Rd:262
                   SEFilterStates.Rd:149
Forkers            README.md:42,52,54
funder             teal.slice-package.Rd:26
Hoffmann           teal.slice-package.Rd:26
programmatically   NEWS.md:21
                   teal-slice.Rmd:95
repo               README.md:9,15,50,54
reproducibility    FilteredData.Rd:174,322,518,526
                   init_filtered_data.Rd:24
                   NEWS.md:93
UI                 DefaultFilteredDataset.Rd:214,216,230
                   FilteredData.Rd:738,782,809,830,854,875,884,922
                   FilteredDataset.Rd:298,300,315,333,345,347,361,383
                   FilterState.Rd:255,265
                   FilterStateExpr.Rd:313,323
                   FilterStates.Rd:179,256,258,301,335
                   MAEFilteredDataset.Rd:185,187,201
                   MAEFilterStates.Rd:86
                   SEFilterStates.Rd:100,137
                   filter-panel.Rmd:151
                   teal-slice.Rmd:48
```

**Pull request merged**